src/components/application_manager/src/resumption/resumption_data_json.cc

```diff
@@ -359,8 +395,10 @@ void ResumptionDataJson::GetDataForLoadResumeData(
   sync_primitives::AutoLock autolock(resumption_lock_);
   smart_objects::SmartObject so_array_data(smart_objects::SmartType_Array);
   int i = 0;
-  for (Json::Value::iterator it = GetSavedApplications().begin();
-       it != GetSavedApplications().end();
+  resumption::LastStateAccessor accessor = last_state_wrapper_->get_accessor();
+  Json::Value dictionary = accessor.GetData().dictionary();
+  Json::Value& saved_apps = GetSavedApplications(dictionary);
+  for (Json::Value::iterator it = saved_apps.begin(); it != saved_apps.end();
        ++it) {
```

Comment:
maybe range-based

Reply:
@SNiukalov fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/f4e32226bda388a572a9edc282e74636a66ad925